### PR TITLE
Red Chapter 20

### PR DESCRIPTION
**Pull request by utopia-2**

### CODE REVIEW

File: red/My-List-Util/t/has_hosts.t [hunk 1]
```diff
@@ -0,0 +1,25 @@
+use Test::More;
+use Test::File;
+
+my $nix_path = '/etc/hosts';
+my $windows_path = 'C:\Windows\System32\drivers\etc\hosts';
+
+my %supported_unixlike = map { $_ => 1 } qw( bsdos darwin linux openbsd freebsd solaris );
```
Comment (alexdeas) [inline red/My-List-Util/t/has_hosts.t:7]: Did you test on all these platforms?

File: red/My-List-Util/lib/Test/My/List/Util.pm [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Verifies the outcome of a list summation is valid.
+
+=cut
+
+sub sum_ok {
+  my ( $actual, $expect, $name ) = @_;
+  $name //= "Sums to $expect";
+
+  if ( !defined $actual || $actual !~ /^-?\d*(?:\.\d+)?(?:e[+-]\d+)?$/ ) {
```
Comment (alexdeas) [inline red/My-List-Util/lib/Test/My/List/Util.pm:51]: Didn't you grief @Spinodal about regexes and weird perl numbers?

Comment (utopia-2) [inline red/My-List-Util/lib/Test/My/List/Util.pm:51]: Yup, I cover the ones I want to cover, but don't consider `nan` or `inf` to have equality (`inf` != `inf` in my books).

File: red/My-List-Util/t/has_hosts.t (hunk 1 above)
Comment (utopia-2) [inline red/My-List-Util/t/has_hosts.t:7]: Nope, just Linux. Didn't want to spend the evening configuring VMs for everything else.

**Pull request merged**